src/pages/blocks/[blockId]/transactions/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { useRouter } from 'next/router'
+import { TableHeader, TableBody } from '../../../../components/Table/index'
+
+interface Transaction {
```

Comment:
Please import these from whale, and don't re-create your own.

Reply:
Sorry this no applicable. The changes for this was deleted in the blocks page. But it wasn't done for this branch.